keystone_api/apps/users/tests/utils.py

```diff
@@ -25,4 +25,4 @@ def create_test_user(
         The saved user account
     """
 
-    return User.objects.create_user(username, first_name, last_name, email, password, **kwargs)
+    return User.objects.create_user(username, password, **kwargs)
```

Comment:
I still don't understand why this line should be changed.

Reply:
I think it was because those positional arguments were not typed as `Optional` in the function signature, so despite not being required by the model, the request was still saying they were required. I'll switch them back to positional with `Optional` and see how that goes instead.

Reply:
Apologies, but I don't understand. 

The `create_test_user` function takes `first_name`, `last_name` an `email` as arguments. The change made here results in those arguments being ignored upon creation of the user object.

What does this have to do an incoming request?

Reply:
Sorry, its because the function returns `User.objects.create_user`, and I was speaking of that function's signature. You are correct that `utils` does not need to change. I switch things back to how they were.